util/csvresponse.py

```diff
@@ -122,13 +124,19 @@ def _create_csv(self, dataset, filehandle):
         """
         # for a glider, get the lat lon
         if self.stream_request.stream_key.is_glider:
-            if GLIDER_GPS_BASED_LAT in dataset.data_vars.keys():
+            if GLIDER_GPS_BASED_LAT in dataset.data_vars.keys() and GLIDER_GPS_BASED_LON in dataset.data_vars.keys():
```

Comment:
Is it preferable to previous behavior to only set the dataset latitude variables if the longitude is also present (and vice versa)?

Reply:
I think so, plus it makes things consistent with how it's being done in jsonresponse.py (in _deployment_particles, under "if stream_key.is_glider") and netcdf_utils.py (in rename_glider_lat_lon, under the inner "if" statements).